<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TeleRunner</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	color: #0088cc;
}

QLabel {
color: black
}

QLabel#telerunner {
color: #0088cc;
border-radius: 10px;
border: 3px solid #0088cc;
background-color: white;
}

QMainWindow {
	background-color: white;
}

QWidget {
	background-color: white;
}

QLineEdit {
	color: #293A4C;

	border: 3px solid #0088cc;
	border-radius: 7px;
	background-color: white
}

QMenuBar {
color: #293A4C
}

QMenuBar::item:selected { /* when selected using mouse or keyboard */
    background: #ebfcff;
}

QAction {
color: #40A7E3;
}

QMenu {
color: #293A4C
}

QMenu::item:selected { /* when user selects item using mouse or keyboard */
    background-color: #ebfcff;
}


QTextBrowser{
border: 3px solid #0088cc;
border-radius: 15px;
color: #0088cc;
padding: 10px;
background-color: white
}

QTextBrowser#consoleStdError {
	border: 3px solid #0088cc;
	border-radius: 15px;
	color: #f57373;
	padding: 10px;
	background-color: white
}

QPushButton {
color: #293A4C;
background-color: white;
border-radius: 5px;
border: 3px solid #0088cc;
}

QPushButton:hover {
	background-color: #ebfcff;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #d4f6fc, stop: 1 #ebfcff);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0" colspan="4">
     <layout class="QHBoxLayout" name="console">
      <item>
       <widget class="QTextBrowser" name="consoleStdOutput">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="consoleStdError"/>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="gitrepoLabel">
      <property name="text">
       <string>Github Repository</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0" colspan="4">
     <layout class="QGridLayout" name="bottomMenu">
      <item row="1" column="1">
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear Console</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="botGitUpdateButton">
        <property name="text">
         <string>Update from Github</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="upTimeLabel">
      <property name="text">
       <string>bot uptime:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="madeBy">
      <property name="text">
       <string>made by vyual | v0.0.1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item row="8" column="2" colspan="2">
     <layout class="QHBoxLayout" name="pathMenu">
      <item>
       <widget class="QLineEdit" name="pathLineEdit">
        <property name="placeholderText">
         <string>Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exploreButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="2" colspan="2">
     <widget class="QLineEdit" name="urlGitRepoEdit">
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Link</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="fliePathLabel">
      <property name="text">
       <string>Run File (python)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="telerunner">
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TeleRunner</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="upTime">
      <property name="text">
       <string>0d:0h:0m:0s</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>914</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionRussian"/>
    <addaction name="actionEnglish"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAuthor"/>
    <addaction name="separator"/>
    <addaction name="actionGithub"/>
   </widget>
   <widget class="QMenu" name="menuEnv_files">
    <property name="title">
     <string>Env files</string>
    </property>
    <addaction name="actionEdit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuAbout"/>
   <addaction name="menuEnv_files"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionRussian">
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="text">
    <string>Author</string>
   </property>
  </action>
  <action name="actionGithub">
   <property name="text">
    <string>Github</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
